packages/front-end/pages/api/init.ts

```diff
@@ -77,8 +78,8 @@ export default function handler(req: NextApiRequest, res: NextApiResponse) {
         ? "disable"
         : "enable",
     sentryDSN: NEXT_PUBLIC_SENTRY_DSN || "",
-    usingSSO: !!SSO_CONFIG,
-    storeSegmentsInMongo: !!STORE_SEGMENTS_IN_MONGO,
+    usingSSO: stringToBoolean(SSO_CONFIG),
```

Comment:
Leave this one as it was. We want the behavior for usingSSO to be "not empty". So setting SSO_CONFIG to "false" should still count as it being set.